doc/nrf/app_dev/device_guides/nrf54h/ug_nrf54h20_custom_pcb.rst

```diff
@@ -75,17 +76,325 @@ This is useful when making a change on the PCB (for example, when changing the c
+
+The resulting JSON configuration is structured as follows:
+
+.. code-block:: json
+
+   {
+     "ioPortPower": {
+       "p1Supply": "EXTERNAL_1V8",
+       "p2Supply": "EXTERNAL_1V8",
+       "p6Supply": "EXTERNAL_1V8",
+       "p7Supply": "EXTERNAL_1V8",
+       "p9Supply": "EXTERNAL_FULL"
+     }
+   }
+
+GPIO power drive and impedance configuration
+--------------------------------------------
+
+This section specifies the IO port impedance settings for **P6** and **P7**.
+
+The nRF54H20 SoC allows you to select from these supported impedance values:
+
+* 33 Ohms
+* 40 Ohms
+* 50 Ohms
+* 66 Ohms
+* 100 Ohms
+
+The configuration can be represented as follows:
+
+.. code-block:: json
+
+   {
+     "ioPortImpedance": {
+       "p6ImpedanceOhms": 50,
+       "p7ImpedanceOhms": 50
+     }
+   }
+
+Low Frequency Oscillator (LFOSC) configuration
+----------------------------------------------
+
+The JSON format is inside ``lfosc: { lfxo: { ... } }``.
+
+ACCURACY
+^^^^^^^^
+
++--------------------+
+| JSON (accuracyPPM) |
++====================+
+| 20                 |
++--------------------+
+| 30                 |
++--------------------+
+| 50                 |
++--------------------+
+| 75                 |
++--------------------+
+| 100                |
++--------------------+
+| 150                |
++--------------------+
+| 250                |
++--------------------+
+| 500                |
++--------------------+
+
+MODE
+^^^^
+
++-------------+
+| JSON (mode) |
++=============+
+| CRYSTAL     |
++-------------+
+| EXT_SINE    |
++-------------+
+| EXT_SQUARE  |
++-------------+
+
+LOADCAP
+^^^^^^^
+
++---------------------------------+
+| JSON (builtInLoadCapacitancePf) |
++=================================+
+| Integer [pF], min 1, max 25     |
++---------------------------------+
+
+.. note::
+   In the JSON configuration, the load capacitance parameter is only applied if the option ``builtInLoadCapacitors`` is explicitly set to true.
+
+TIME (LFXO startup time)
+^^^^^^^^^^^^^^^^^^^^^^^^
+
++-----------------------------+
+| JSON (startupTimeMs)        |
++=============================+
+| Integer [ms], min 1, max 25 |
++-----------------------------+
+
+See the following example:
+
+.. code-block:: json
+
+   {
+     "lfosc": {
+       "source": "LFXO",
+       "lfxo": {
+         "mode": "CRYSTAL",
+         "accuracyPPM": 20,
+         "startupTimeMs": 600,
+         "builtInLoadCapacitancePf": 15,
+         "builtInLoadCapacitors": true
+       }
+     }
+   }
+
+Source
+^^^^^^
+
+The ``source`` option can be one of the following:
+
+* ``LFXO``, when an external crystal oscillator is in place.
+* ``LFRC``, when an external Crystal Oscillator is not in place.
+
+This means that the device can use either ``LFRC`` or ``SYNTH`` as clock sources.
+
+LFRC autocalibration configuration
+----------------------------------
+
+The three values provided correspond to the LFRC autocalibration configuration:
```

Comment:
```suggestion
The following parameters are used in the LFRC autocalibration configuration:
```
Or something like this. There are no provided values here, so referring to such is off.